src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMutation.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    internal class CollectImpl : CollectFunction, IAsyncTexlFunction
```

Comment:
>IAsyncTexlFunction [](http://example.com/codeflow?start=50&length=18)

Since we are unifying Collect(), do we need IAsyncTexlFunction? or now we can go through our regular pipeline? (Appending implementation in _library)

Reply:
Hosts will still need to enable mutation functions explicitly. That's why we are not including them in _library.

Reply:
Ahh makes sense